src/java.desktop/share/classes/com/sun/media/sound/SoftSynthesizer.java

```diff
@@ -634,15 +632,19 @@ public boolean remapInstrument(Instrument from, Instrument to) {
         }
     }
 
+   static interface RunnableAction<T> {
```

Comment:
Can we use `.java.util.functionSupplier`  instead?

Reply:
I could, but I chose to do this instead.